isso/db/comments.py

```diff
@@ -217,16 +217,16 @@ def fetchall(self, mode=5, after=0, parent='any', order_by='id',
 
         # custom sanitization
         if order_by not in ['id', 'created', 'modified', 'likes', 'dislikes', 'tid']:
-            sql.append('ORDER BY ')
-            sql.append("comments.created")
+            sql.append('ORDER BY CASE WHEN comments.parent IS NOT NULL THEN comments.id END, ')
```

Comment:
This first `if` case is already the fallback for when no sorting is specified. No need to alter anything here, instead leave `comments.created` forsorting.

Reply:
Sure thing, I'll switch to it and make another commit when I get off of work in a few hours.

Reply:
Switched to it (sorry this took so long, had some server nonsense pop up.